js/frontend/src/pages/index.astro

```diff
@@ -1,7 +1,7 @@
 ---
 import Layout from '@/layouts/Layout.astro';
 // Component Imports
-import { NewUrlForm } from '@/components/NewUrlForm.tsx';
```

Comment:
Tbh I don't understand how this was working with a broken import? Does the astro compiler add any magic to the `src/components` directory?